3574/1/top_gems.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'info_parser'
+require_relative 'top_function'
+require_relative 'file_reader'
+require_relative 'output'
+require_relative 'options'
+
+HEADER = %i[name_id used_by watch stars fork issues contributors].freeze
```

Comment:
please move these constants to files where they are used
please note that they should be inside classes